framework/include/interfaces/Coupleable.h

```diff
@@ -1458,6 +1505,10 @@ class Coupleable
   const bool _is_fv;
 
   const MooseObject * const _obj;
+
+  /// vector tag names for which we need to request older solution states from the system
+  const std::set<std::string> _older_state_tags = {Moose::OLD_SOLUTION_TAG,
```

Comment:
I added these in order to be able to recognize tags for which we would need to request additional solution states from the `SystemBase`. Otherwise a user who asks for an old/older tag may get an error that the vector tag does not exist in the subproblem/system

Reply:
Did you mean a user explicitly ask for an old/older tag? I would imagine these tags will be required by transient executioners or/and time integrators.

Reply:
those requests can happen too late